packages/core/src/domain/document/upload.ts

```diff
@@ -18,3 +19,16 @@ export function createUploadMetadataFilePath(
   const uploadFilePath = createUploadFilePath(cxId, patientId, docName);
   return `${uploadFilePath}_metadata.xml`;
 }
+
+export function createAttachmentUploadPath({
```

Comment:
this may be a larger issue, but this is called `createAttachmentUploadPath` which implies it's producing the "full" S3 key (i.e. /x/y/z/filename)

but it's returning ``${fileName}_${ownId}${extension}`;` which "looks" like just a modified file name with no path.

If we use fileName to include the full path, then I'd rename this to be `createAttachmentUploadFileName`.

Reply:
good point!